### ISSUE TITLE
[Bug] Xamarin Forms FormattedText MaxLines not working on Android

### ISSUE THREAD

**Issue by thomashoef** [xamarin/Xamarin.Forms#9149]
### Description
The MaxLines property has no effect on a Label with FormattedText on Android. If MaxLines > 1 the label on Android renders only 1 line with middle truncation even though the formatted text contains 10 lines worth of text. On iOS and UWP the label renders the 10 lines correctly.

### Steps to Reproduce

1. Create a label with formatted text and set maxlines to a value > 1

```
    <Label Grid.Row="1" Grid.Column="0" Margin="0,8,0,0" Grid.ColumnSpan="{Binding LayoutSettings.DescriptionColumnSpan}"
                       FormattedText="{Binding LayoutSettings.FormattedText}" 
                       FontSize="{forms:Platform SmallText}" MaxLines="100"
                       AutomationId="detailsText"/>
```

2. Check the label rendered on iOS and UWP and compare to Android 
3. On Android the label renders on a single line with middle truncation

### Expected Behavior
Android renders the label text adhering to the maxLines value set in the XAML

### Actual Behavior
On Android the label renders on a single line with middle truncation

### Basic Information

- Version with issue: 4.4.0.991265
- Last known good version: ?
- IDE: Rider
- Platform Target Frameworks: 
  - Android: 9.0
  - Android Support Library Version: 28.0.3

**Comment by kingces95** [comment]
Could you please attach a zipped reproduction to the bug? Thanks!

**Comment by hartez** [comment]
@thomashoef I'm trying to reproduce the problem you describe, but so far I'm not having any luck. Here's my attempted repro:

[_9149 Repro.zip](https://github.com/xamarin/Xamarin.Forms/files/4051809/_9149.Repro.zip)

It seems to work fine - is there a detail I'm missing?

**Comment by thomashoef** [comment]
Hi @hartez , thank you for checking this. I tested in your sample and could not reproduce the issue. I noticed thought that my Xam Forms version was still at 4.3.0.991211. I could not immediately get the project to run after updating to your (latest) version. I then added LineBreakMode="NoWrap" in my project, and guess what, the maxlines were respected after adding this :). I have tested my layout in your sample etc. and it worked, so I will upgrade Xam Forms again and I assume that this will fix the issue on Droid. Thanks!
